load-test-framework/src/main/java/com/google/pubsub/flic/Driver.java

```diff
@@ -141,6 +167,11 @@
           "message loss. If set less than 1, this flag is ignored."
   )
   private int numberOfMessages = 0;
+  
+  private final String SPREADSHEET_ID = System.getenv("LOADTEST_SPREADSHEET_ID");
+  private final String APPLICATION_NAME = "loadtest-framework";
+  private final String DATA_STORE_DIR = 
+      System.getProperty("user.home") + "/.credentials/sheets.googleapis.com-loadtest-framework";
```

Comment:
Fixed. I did keep this location set to be the default rather than empty string, because it still requires the feature being turned on by a user by setting spreadsheet_id. This being set to some default won't impact user experience unless they choose to use the feature.